services/frontend-service/pkg/handler/release.go

```diff
@@ -164,7 +164,7 @@ func (s Server) HandleRelease(w http.ResponseWriter, r *http.Request, tail strin
 				}
 				if !validSignature {
 					w.WriteHeader(400)
-					fmt.Fprintf(w, "signature not found for %s", environmentName)
+					fmt.Fprintf(w, "signature not found or invalid for environment %s", environmentName)
```

Comment:
```suggestion
					fmt.Fprintf(w, "signature is invalid or was not found for environment %s", environmentName)
```
NIT: This sounds a bit more natural
Or maybe "invalid signature or it was not found for environment %s"?